examples/matRad_example13_fitAnalyticalParticleBaseData.m

```diff
@@ -124,25 +128,44 @@
 pln.propOpt.runDAO          = false;  % 1/true: run DAO, 0/false: don't / will be ignored for particles
 pln.propOpt.runSequencing   = false;  % 1/true: run sequencing, 0/false: don't / will be ignored for particles and also triggered by runDAO below
 
+% retrieve scenarios for dose calculation and optimziation
+pln.multScen = matRad_multScen(ct,'nomScen');
+
+
+quantityOpt   = 'RBExD';            % either  physicalDose / effect / RBExD
+modelName     = 'constRBE';         % none: for photons, protons, carbon                                    constRBE: constant RBE model
```

Comment:
Fitting with constRBE does not make sense. I would rather set quantityOpt pyhsicalDose & modelname none

Reply:
Done.